Networking/Networking/Model/Country.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import Codegen
+
+/// Represents a Country Entity.
+///
+public struct Country: Decodable, Equatable, GeneratedFakeable {
+    public let code: String
```

Comment:
Optional. Add an example as a comment, e.g.

// ISO-3166 two letter country code, e.g. CA for Canada